application/src/main/java/org/togetherjava/tjbot/features/moderation/NoteCommand.java

```diff
@@ -51,10 +51,13 @@ public NoteCommand(ModerationActionsStore actionsStore) {
 
     @Override
     public void onSlashCommand(SlashCommandInteractionEvent event) {
-        OptionMapping targetOption = event.getOption(USER_OPTION);
-        Member author = event.getMember();
-        Guild guild = event.getGuild();
-        String content = event.getOption(CONTENT_OPTION).getAsString();
+        OptionMapping targetOption =
+                Objects.requireNonNull(event.getOption(USER_OPTION), "The user is null");
+        Member author = Objects.requireNonNull(event.getMember());
+        Guild guild = Objects.requireNonNull(event.getGuild());
+        String content =
+                Objects.requireNonNull(event.getOption(CONTENT_OPTION), "The content is null")
+                    .getAsString();
```

Comment:
```suggestion
        OptionMapping targetOption = event.getOption(USER_OPTION);
        Member author = Objects.requireNonNull(event.getMember());
        Guild guild = Objects.requireNonNull(event.getGuild());
        String content = event.getOption(CONTENT_OPTION).getAsString();
```

restore `targetOption` and `content` as discussed in discord, or don't idek